Extension/src/LanguageServer/extension.ts

```diff
@@ -1379,3 +1382,88 @@ export async function getIncludes(maxDepth: number): Promise<any> {
     const includes = await clients.ActiveClient.getIncludes(maxDepth);
     return includes;
 }
+
+// This uses several workarounds for interacting with the hover feature.
+// A proposal for dynamic hover content would help, such as the one here (https://github.com/microsoft/vscode/issues/195394)
+async function onCopilotHover(): Promise<void> {
+    if (!vscode.window.activeTextEditor) { return; }
```

Comment:
Does our linter allow this? I expected that it would want:
```ts
if (!vscode.window.activeTextEditor) {
    return;
}
```

Reply:
@bobbrow Our formatter recently changed and allows this currently. Maybe there's some setting we need to change to alter it

Reply:
I can look into it later. I'm not a fan of this style.

Reply:
I meant to update this to match the rest of the formatting, I'll update it now. I don't have any strong preference for one or the other.